packages/common/src/env.ts

```diff
@@ -14,9 +14,9 @@ const env: Env = {
 };
 
 export const configureEnv = (options: Partial<Env>) => {
-  env.APP_VERSION ||= options.APP_VERSION;
-  env.APP_API_BASE_URL ||= options.APP_API_BASE_URL;
-  env.APP_PLAYER_ID ||= options.APP_PLAYER_ID;
+  env.APP_VERSION = options.APP_VERSION || env.APP_VERSION;
+  env.APP_API_BASE_URL = options.APP_API_BASE_URL || env.APP_API_BASE_URL;
+  env.APP_PLAYER_ID = options.APP_PLAYER_ID || env.APP_PLAYER_ID;
```

Comment:
I've reset this, because it can try to set to `undefined`, which is not allowed by the typings.

Reply:
I also did that in my PR 😄

Reply:
I saw :D